libs/webgui/pilight.js

```diff
@@ -353,6 +361,102 @@ function createPendingSwitchElement(sTabId, sDevId, aValues) {
+		}
+		$("div").trigger("create");
+		$('#'+sDevId+'_contact_closed').checkboxradio();
+		$('#'+sDevId+'_contact_opened').checkboxradio();
+		$('#'+sDevId+'_contact_closed').bind("change", function(event, ui) {
+			event.stopPropagation();
+			if('confirm' in aValues && aValues['confirm']) {
+				if(window.confirm("Are you sure?") == false) {
+					return false;
+				}
+			}
+			i = 0;
+			oLabel = this.parentNode.getElementsByTagName('label')[0];
+			$(oLabel).removeClass('ui-btn-active');
+			x = window.setInterval(function() {
+				i++;
+				if(i%2 == 1)
+					$(oLabel).removeClass('ui-btn-active');
+				else
+					$(oLabel).addClass('ui-btn-active');
+				if(i==2)
+					window.clearInterval(x);
+			}, 100);
+
+			if(oWebsocket) {
+				if('all' in aValues && aValues['all'] == 1) {
+					var json = '{"action":"control","code":{"device":"'+sDevId+'","state":"'+this.value+'","values":{"all": 1}}}';
+				} else {
+					var json = '{"action":"control","code":{"device":"'+sDevId+'","state":"'+this.value+'"}}'
+				}
+				oWebsocket.send(json);
+			} else {
+				bSending = true;
+				if('all' in aValues && aValues['all'] == 1) {
+					$.get(sHTTPProtocol+'://'+location.host+'/control/control?device='+sDevId+'&state='+this.value+'&values[all]=1');
+				} else {
+					$.get(sHTTPProtocol+'://'+location.host+'/control?device='+sDevId+'&state='+this.value);
+				}
+				window.setTimeout(function() { bSending = false; }, 1000);
+			}
+		});
+		$('#'+sDevId+'_contact_opened').bind("change", function(event, ui) {
+			event.stopPropagation();
+			if('confirm' in aValues && aValues['confirm']) {
+				if(window.confirm("Are you sure?") == false) {
+					return false;
+				}
+			}
+			i = 0;
+			oLabel = this.parentNode.getElementsByTagName('label')[0];
+			$(oLabel).removeClass('ui-btn-active');
+			x = window.setInterval(function() {
+				i++;
+				if(i%2 == 1)
+					$(oLabel).removeClass('ui-btn-active');
+				else
+					$(oLabel).addClass('ui-btn-active');
+				if(i==2)
+					window.clearInterval(x);
+			}, 100);
+
+			if(oWebsocket) {
+				if('all' in aValues && aValues['all'] == 1) {
+					var json = '{"action":"control","code":{"device":"'+sDevId+'","state":"'+this.value+'","values":{"all": 1}}}';
+				} else {
+					var json = '{"action":"control","code":{"device":"'+sDevId+'","state":"'+this.value+'"}}'
+				}
+				oWebsocket.send(json);
+			} else {
+				bSending = true;
+				if('all' in aValues && aValues['all'] == 1) {
+					$.get(sHTTPProtocol+'://'+location.host+'/control?device='+sDevId+'&state='+this.value+'&values[all]=1');
+				} else {
+					$.get(sHTTPProtocol+'://'+location.host+'/control?device='+sDevId+'&state='+this.value);
+				}
+				window.setTimeout(function() { bSending = false; }, 1000);
+			}
+		});
+	}
+	$('#'+sDevId+'_contact_opened').checkboxradio('disable');
+	$('#'+sDevId+'_contact_closed').checkboxradio('disable');
+	$('#'+sDevId+'_contact_opened').fadeTo(0,0.1);
```

Comment:
removed. opacity is a function of the browser used for showing elements as being disabled.